test/services/garbageCollector.test.js

```diff
@@ -64,6 +63,7 @@ describe('Test: GarbageCollector service', () => {
           clock.tick(oneDay);
 
           should(kuzzle.pluginsManager.trigger).called();
+          return null;
```

Comment:
why this ?

Reply:
To prevent bluebird from complaining about the promise chain not returning anything

Reply:
ok